src/io/flutter/ProjectOpenActivity.java

```diff
@@ -56,6 +59,7 @@ public void runActivity(@NotNull Project project) {
       // We can't do anything without a Flutter SDK.
       return;
     }
+    excludeAndroidFrameworkDetector(project);
```

Comment:
Do we want to do this every time we open the project, or once per project? We do do w few things just once per project (showing the 'flutter has hot reload' notification) - this might be a candidate for the same.

Reply:
It doesn't appear to be stored in project files.